app.js

```diff
@@ -5,13 +5,16 @@ const fs = require('fs')
 const path = require('path')
```

Comment:
Could/should this file move into `app` as well?

Reply:
If we did this, it might then make sense to put all of the views under `app/views`?

Reply:
Yep, you're right both should be moved. They were in the file structure, but I rushed into publishing the branch.